### DISCUSSION TITLE
FirebaseFirestore naming conventions

### DISCUSSION THREAD

**Discussion by dflorey** [firebase/firebase-js-sdk#4561] [General]
Is there a reason why you've picked the class name FirebaseFirestore instead of just Firestore (and FirebaseApp instead of just App)?
It feels a bit inconsistent to me as the other services are just called "Functions" etc. (I'd prefer shorter names)

**Comment by akauppi** [comment]
Likewise. 

Also, when discussing naming please keep in mind that ES modules allows renaming on import: 

```
import { abc as ABC } from ‘abc’
```

I consider that only as first-aid, in case there are collisions with package names.

**Comment by akauppi** [comment]
Would like to see Firebase authors respond to this. It looks like the most easy-picking improvement in the Discussions.

While I'm kind of fine with any naming since it gets buried in application specific wrapping, these names show up also in the IDE syntax highlighting and browser debugging (was trying to find a screenshot but failed for now), so I support the shorter names. However, consistency is probably the #1 reason.
